src/views/tickets/email.php

```diff
@@ -286,21 +286,32 @@
 						$venue_address_tag = 'a';
 						$venue_address_style .= ' color:#006caa !important; text-decoration:underline;';
 					}
-
 				}
 
-				$start_date = null;
+				$event_date = null;
 
 				/**
-				 * Filters whether or not the event start date should be included in the ticket email
+				 * Filters whether or not the event date should be included in the ticket email.
+				 *
+				 * @since TBD
+				 *
+				 * @var bool Include event date? Defaults to false.
+				 * @var int  Event ID
+				 */
+				$include_event_date = apply_filters( 'event_tickets_email_include_event_date', false, $event->ID );
```

Comment:
Let's take this opportunity to switch the filter name to our event-tickets naming convention:

```
tribe_tickets_email_include_event_date
```